src/qt/utilitydialog.h

```diff
@@ -34,6 +36,8 @@ class HelpMessageDialog : public QDialog
 
 private Q_SLOTS:
     void on_okButton_accepted();
+    void setAboutWindowTitle(const NetworkStyle *networkStyle = nullptr);
+    void setChainTypeIconOnAboutLogo(const NetworkStyle *networkStyle = nullptr);
```

Comment:
Why these two member functions are declared as slots?

Reply:
Sorry, that's wrong, I meant to put them just on the private section. I'll fix it soon. Thanks!

Reply:
done!